conda/gateways/logging.py

```diff
@@ -208,7 +208,7 @@ def set_all_logger_level(level=DEBUG):
 @cache
 def set_file_logging(logger_name=None, level=DEBUG, path=None):
     if path is None:
-        timestamp = datetime.utcnow().strftime("%Y%m%d-%H%M%S")
+        timestamp = datetime.now(datetime.UTC).strftime("%Y%m%d-%H%M%S")
```

Comment:
[`datetime.utcnow` deprecated in 3.12](https://docs.python.org/3/library/datetime.html#datetime.datetime.utcnow):

<img width="1660" height="576" alt="Image" src="https://github.com/user-attachments/assets/a7885f7d-1bd2-4d95-9359-831ca319b011" />